### ISSUE TITLE
elasticsearch + odoo

### ISSUE THREAD

**Issue by kkoci** [camptocamp/odoo-elasticsearch-kibana#4]
Hi, I'm just curious to know if you're going to push some modules with this integration anytime soon!

Thanks in advance!

**Comment by guewen** [comment]
Actually you can find my pull request here https://github.com/camptocamp/odoo-elasticsearch-kibana/pull/3 :-)

**Comment by guewen** [comment]
(And we use it with https://github.com/OCA/server-tools/pull/245 for creating the SQL views)

**Comment by kkoci** [comment]
Thank you very much @guewen just one more question, is this just for Odoo v7?

**Comment by guewen** [comment]
Yes actually we only needed them for Odoo 7.0. Should not be hard to make them work on 8.0 though.

**Comment by kkoci** [comment]
Yes , sure, well thank you very much :) , maybe I can port them to v8 and let you know.

Cheers!
